tests/robot-tests/tests/admin/bau/create-amendment-check-permalink.robot

```diff
@@ -0,0 +1,198 @@
+*** Settings ***
+Resource  ../../libs/admin-common.robot
+Resource  ../../libs/common.robot
+Library  ../../libs/admin_api.py
+Library  Collections
+
+Force Tags  Admin  Local  Dev  AltersData
+
+Suite Setup       Run Keywords  user signs in as bau1
+Suite Teardown    Run Keywords  user closes the browser
+
+*** Variables ***
+${DETAILS_HEADING}  Academic Year 2020/21 (not Live)
+${AMEND_RELEASE_HEADING}  Academic Year 2020/21 (Live - Latest release)
+${PUBLICATION_NAME}  Permalink Test Publication %{RUN_IDENTIFIER}
+${TOPIC_NAME}  %{TEST_TOPIC_NAME}
+${SUBJECT_NAME}   Dates test subject
+
+*** Test Cases ***
+create publication & releae
+  ${PUBLICATION_ID}=  user creates test publication via api  ${PUBLICATION_NAME}
+  user create test release via api  ${PUBLICATION_ID}  AY  2020
+
+upload a subject
+     user navigates to release summary from admin dashboard  ${PUBLICATION_NAME}  ${DETAILS_HEADING}
+     user clicks link  Data and files
+     user waits until page contains element  css:#dataFileUploadForm-subjectTitle
+     user enters text into element  css:#dataFileUploadForm-subjectTitle   ${SUBJECT_NAME}
+     user chooses file   css:#dataFileUploadForm-dataFile       ${CURDIR}${/}files${/}dates.csv
+     user chooses file   css:#dataFileUploadForm-metadataFile   ${CURDIR}${/}files${/}dates.meta.csv
+     user clicks button  Upload data files
+     user waits until h2 is visible  Uploaded data files
+     user waits until page contains accordion section   ${SUBJECT_NAME}
+     user opens accordion section   ${SUBJECT_NAME}
+     ${section}=  user gets accordion section content element  ${SUBJECT_NAME}
+     user checks headed table body row contains  Status           Complete  ${section}  180
+
+Go to "Release status" page
+    [Tags]  HappyPath
+    user clicks link   Release status
+    user waits until h2 is visible  Release status
+    user waits until page contains button  Edit release status
+
+Approve release
+    [Tags]  HappyPath
+    user clicks button  Edit release status
+    user waits until h2 is visible  Edit release status
+    user clicks radio   Approved for publication
+    user enters text into element  id:releaseStatusForm-internalReleaseNote  Approved by UI tests
+    user clicks radio   As soon as possible
+    user clicks button   Update status
+
+Wait for release process status to be Complete
+    [Tags]  HappyPath
+    user waits for release process status to be  Complete    ${release_complete_wait}
+    user reloads page  # EES-1448
+    user checks page does not contain button  Edit release status
+
+Go to Table Tool page
+    [Tags]  HappyPath
+    environment variable should be set  PUBLIC_URL
+    user goes to url  %{PUBLIC_URL}/data-tables
+    user waits until h1 is visible  Create your own tables online
+
+Select "Pupil Exclusions" publication
+    [Tags]  HappyPath
+    user opens details dropdown    Test theme
```

Comment:
You can replace `Test theme` with `%{TEST_THEME_NAME}`. Can do this elsewhere if necessary